crawlerr/services/crawler_service.py

```
@@ -442,6 +393,377 @@ async def get_sitemap_urls(self, base_url: str) -> List[str]:
+            for crawl_result in crawl_results:
+                if crawl_result.success:
+                    page_content = self._process_crawl_result(crawl_result, extract_raw_html=False)
+                    pages.append(page_content)
+                    stats.total_pages_crawled += 1
+                else:
+                    stats.total_pages_failed += 1
+                    logger.warning(f"Failed to crawl {crawl_result.url}: {crawl_result.error_message}")
+                    
+        return pages
+    
+    def _process_crawl_result(self, crawl_result, extract_raw_html: bool = False) -> PageContent:
```

Ruff (0.12.2)

Missing type annotation for function argument `crawl_result`

(ANN001)